### PR TITLE
fix: disable Debugger defaults in unsupported regions

### PR DESCRIPTION
**Pull request by knakad**
*Description of changes:* disable Debugger defaults in unsupported regions

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [X] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [X] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [X] I have used the regional endpoint when creating S3 and/or STS clients (if appropriate)
- [X] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [X] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [X] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [X] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker/fw_utils.py [hunk 1]
```diff
@@ -504,3 +505,16 @@ def python_deprecation_warning(framework, latest_supported_version):
     return PYTHON_2_DEPRECATION_WARNING.format(
         framework=framework, latest_supported_version=latest_supported_version
     )
+
+
+def _region_supports_debugger_feature(region_name):
```
Comment (laurenyu) [inline src/sagemaker/fw_utils.py]: [nit] I think "feature" is redundant in both the code and prose

File: src/sagemaker/fw_utils.py [hunk 2]
```diff
@@ -55,6 +55,7 @@
 VALID_PY_VERSIONS = ["py2", "py3"]
 VALID_EIA_FRAMEWORKS = ["tensorflow", "tensorflow-serving", "mxnet", "mxnet-serving"]
 VALID_ACCOUNTS_BY_REGION = {"us-gov-west-1": "246785580436", "us-iso-east-1": "744548109606"}
+DEBUGGER_FEATURE_UNSUPPORTED_REGIONS = ["us-gov-west-1", "us-iso-east-1"]
 ASIMOV_VALID_ACCOUNTS_BY_REGION = {"us-iso-east-1": "886529160074"}
```
Comment (laurenyu) [inline src/sagemaker/fw_utils.py]: it feels weird to split up `VALID_ACCOUNTS_BY_REGION` and `ASIMOV_VALID_ACCOUNTS_BY_REGION`. I'd actually move this new constant to be separate from all the constants that exist for determining a Docker image URI, as this new one has nothing to do with that

File: src/sagemaker/fw_utils.py (hunk 1 above)
Comment (knakad) [reply src/sagemaker/fw_utils.py]: +1. Removed "feature"

File: src/sagemaker/fw_utils.py (hunk 2 above)
Comment (knakad) [reply src/sagemaker/fw_utils.py]: +1. Fixed

**Pull request merged**